source/features/unreleased-commits.tsx

```diff
@@ -91,8 +91,8 @@ async function createLink(
 			href={buildRepoURL('compare', `${latestTag}...${await getDefaultBranch()}`)}
 			aria-label={label}
 		>
-			<TagIcon className="v-align-middle"/>
-			{aheadBy === undeterminableAheadBy || <sup className="ml-n2"> +{aheadBy}</sup>}
```

Comment:
n2 was intentionally to bring it closer together, it's not a label but it's "part of the icon"

Reply:
Only remove `v-align-middle`:

<img width="175" alt="Screenshot 2024-06-30 at 2 20 55 PM" src="https://github.com/refined-github/refined-github/assets/44045911/a04cf614-29e3-4219-910d-1800e2215304">

Ok-ish but in my opinion we should have a little space here.